doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -1049,6 +1048,72 @@ div.testimonial span.testimonial-author p {
   color: #808080;
 }
 
+div.testimonial p {
+  color: #1c1c1c;
+}
+
+/* Installation quickstart */
+
+/* style input radio and checkbox */
+
+div.install > input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  opacity: 0;
+}
+
+/* Style the button */
+div.install > label {
+  display: inline-block;
+  margin-top: 12px;
+  padding: 5px 11px;
+  background-color: #fff3cd;
+  border: none;
+  border-radius: 3px;
+  color: black;
+}
+
+/* Style the button when the checkbox is checked */
+div.install > input:checked + label {
+  background-color: #ff9c34;
+  color: white;
+}
+
+/* Hide expandable content by default */
+.sk-expandable {
+  display: none;
+  background: #ddd;
+}
+
```

Comment:
Since this expandable / quickstart CSS rules are heavily inspired by the spaCy quickstart page, I think we should:

- ask spaCy authors (ping @ines @honnibal) if they are fine with us being "very inspired" by their work :)
- if so add some credits as a comment in our CSS file.

Reply:
Either CSS or HTML or both.

Reply:
I've added a line in both, waiting for feedback.

Reply:
We got the idea for the installation widget from PyTorch initially, so definitely glad to have it spreading! If you haven't seen it, Ines has a repo for the component: https://github.com/ines/quickstart . You don't necessarily have to use the component from the repo of course, but it's probably the right thing to reference in the comments.

Reply:
Thanks for your answer, @honnibal. I've added the ref to the original Ines implementation in the code.